doc/sphinx/Pacemaker_Explained/operations.rst

```diff
@@ -38,113 +38,154 @@ two operations for the same resource with the same name and interval.
 Operation Properties
 ####################
 
-Operation properties may be specified directly in the ``op`` element as
-XML attributes, or in a separate ``meta_attributes`` block as ``nvpair`` elements.
-XML attributes take precedence over ``nvpair`` elements if both are specified.
+An operation property may be specified in any of these ways, from highest
+precedence to lowest:
```

Comment:
It might be worth mentioning the special fence action timeouts somewhere in here

Reply:
Definitely, but that's probably worth a project of its own

Reply:
Might as well add a brief note now